lms/static/sass/application.scss.mako

```diff
@@ -28,72 +28,7 @@
   @import '${env.get('THEME_NAME')}';
 % endif
 
-@import 'base/base';
```

Comment:
NOTE: sadly, we cannot abstract more out of this compile list due to theming configuration still being controlled through Mako logic. The rest need to stay or I can create more ``_build-*.scss`` files, but that seems less clear and more messy.